hal/src/subghz/stats.rs

```diff
@@ -152,7 +152,7 @@ impl core::fmt::Display for Stats<FskStats> {
 
 #[cfg(test)]
 mod test {
-    use crate::subghz::{CmdStatus, LoRaStats, Stats, StatusMode};
+    use super::{CmdStatus, LoRaStats, Stats, StatusMode};
```

Comment:
I think this needs to be `super::super` because it is another module deep, other than that looks good to me!